src/cryptonote_core/blockchain.h

```diff
@@ -1427,16 +1440,6 @@ namespace cryptonote
     bool check_block_timestamp(std::vector<uint64_t>& timestamps, const block& b, uint64_t& median_ts) const;
     bool check_block_timestamp(std::vector<uint64_t>& timestamps, const block& b) const { uint64_t median_ts; return check_block_timestamp(timestamps, b, median_ts); }
 
-    /**
-     * @brief get the "adjusted time"
-     *
-     * Currently this simply returns the current time according to the
-     * user's machine.
-     *
-     * @return the current time
-     */
-    uint64_t get_adjusted_time() const;
-
```

Comment:
Why was this declaration moved up 400 lines? New spot seems less appropriate based on other declarations nearby.

Reply:
To make it public.

Reply:
I guess I could add public: and private: and keep it where it was though. Seemed a bit meh but it minimizes the diff...

Reply:
Ah if it's being made public that makes sense.